.github/workflows/ci.yaml

```diff
@@ -119,7 +119,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9, 3.10-dev]
+        python-version: [3.6, 3.7, 3.8, 3.9, "3.10", "3.11-dev"]
```

Comment:
I doscovered the place to check for available version : https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json

Reply:
> I doscovered the place to check for available version : https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json

👍🏻 Can you try not to mark discussions as resolved shortly after you commented on them? It isn't an issue here, but sometimes it can be difficult to find the comment.

Reply:
Sorry the discussion is auto-resolved by github when applying a suggestion.

Reply:
😅 Didn't though about that